src/utils/VerificationValidationParser.cpp

```diff
@@ -153,9 +156,62 @@ void Parser::searchFinalDefense(Result* r) {
     }
 }
 
-Result* Parser::lc(const QString* terminalOutput) {
-    return nullptr; // TODO: implement
+Result* Parser::lc(const QString* cmd, const QString* terminalOutput) {
+	Result final;
+	final.terminalOutput = *terminalOutput;
```

Comment:
this is okay, but ideally trim it (for display purposes, plus saves a bit of memory)
e.g.: set it to terminalOutput.trimmed()